### PR TITLE
Increase test coverage of editCommandTest

### PR DESCRIPTION
**Pull request by cocoanautz**

### CODE REVIEW

File: src/test/java/seedu/address/testutil/TypicalCompanies.java [hunk 1]
```diff
@@ -75,7 +75,7 @@ private TypicalCompanies() {} // prevents instantiation
     /**
      * Returns an {@code AddressBook} with all the typical companies.
```
Comment (AjayArvind2207) [inline src/test/java/seedu/address/testutil/TypicalCompanies.java:76]: Change javadoc to internbook

File: src/test/java/seedu/address/logic/commands/EditCommandTest.java [hunk 2]
```diff
@@ -1,49 +1,90 @@
...
      * Edit filtered list where index is larger than size of filtered list,
-     * but smaller than size of address book
+     * but smaller than size of internbook
      */
     @Test
-    public void execute_invalidPersonIndexFilteredList_failure() {
+    public void execute_invalidCompanyIndexFilteredList_failure() {
         showPersonAtIndex(model, INDEX_FIRST_COMPANY);
         Index outOfBoundIndex = INDEX_SECOND_COMPANY;
         // ensures that outOfBoundIndex is still in bounds of address book list
```
Comment (AjayArvind2207) [inline src/test/java/seedu/address/logic/commands/EditCommandTest.java:90]: address book can be replaced by intern book

**Review by AjayArvind2207 — COMMENTED** [review]
LGTM on the whole except some javadoc issues!

File: src/test/java/seedu/address/logic/commands/ClearCommandTest.java [hunk 3]
```diff
@@ -22,8 +22,8 @@ public void execute_emptyAddressBook_success() {
 
     @Test
     public void execute_nonEmptyAddressBook_success() {
```
Comment (jasperng-nus) [inline src/test/java/seedu/address/logic/commands/ClearCommandTest.java:24]: i think the name here can be changed to InternBook to fit our context

File: src/test/java/seedu/address/logic/commands/ClearCommandTest.java [hunk 4]
```diff
@@ -22,8 +22,8 @@ public void execute_emptyAddressBook_success() {
...
-        Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
-        Model expectedModel = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+        Model model = new ModelManager(getTypicalInternBook(), new UserPrefs());
+        Model expectedModel = new ModelManager(getTypicalInternBook(), new UserPrefs());
         expectedModel.setAddressBook(new InternBook());
```
Comment (jasperng-nus) [inline src/test/java/seedu/address/logic/commands/ClearCommandTest.java:27]: i think this as well, maybe can change the instance of addressbook in the Model Class

File: src/test/java/seedu/address/logic/commands/ListCommandTest.java [hunk 5]
```diff
@@ -22,7 +22,7 @@ public class ListCommandTest {
 
     @BeforeEach
     public void setUp() {
-        model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+        model = new ModelManager(getTypicalInternBook(), new UserPrefs());
         expectedModel = new ModelManager(model.getAddressBook(), new UserPrefs());
```
Comment (jasperng-nus) [inline src/test/java/seedu/address/logic/commands/ListCommandTest.java:26]: i think you could maybe change the instances of addressbook in the ModelManager class to fit our context

File: src/test/java/seedu/address/logic/commands/SortCommandTest.java [hunk 6]
```diff
@@ -14,7 +14,7 @@
 
 public class SortCommandTest {
     private Model model = new ModelManager(getUnsortedTypicalAddressBook(), new UserPrefs());
```
Comment (jasperng-nus) [inline src/test/java/seedu/address/logic/commands/SortCommandTest.java:16]: for this as well, perhaps could change the instances of getUnsortedTypicalAddressBook() to fit our context

File: src/test/java/seedu/address/storage/JsonInternBookStorageTest.java [hunk 7]
```diff
@@ -63,7 +63,7 @@ public void readInternBook_invalidAndValidCompanyInternBook_throwDataLoadingExce
     @Test
     public void readAndSaveInternBook_allInOrder_success() throws Exception {
         Path filePath = testFolder.resolve("TempAddressBook.json");
```
Comment (jasperng-nus) [inline src/test/java/seedu/address/storage/JsonInternBookStorageTest.java:65]: maybe the name of the TempAddressBook.json could be changed to TempInternBook.json but you would need to probably change other instances of this as well, and i am unsure are we allowed to change the json file, maybe we can ask the group

File: src/test/java/seedu/address/storage/JsonInternBookStorageTest.java [hunk 8]
```diff
@@ -63,7 +63,7 @@ public void readInternBook_invalidAndValidCompanyInternBook_throwDataLoadingExce
...
-        InternBook original = getTypicalAddressBook();
+        InternBook original = getTypicalInternBook();
         JsonAddressBookStorage jsonAddressBookStorage = new JsonAddressBookStorage(filePath);
```
Comment (jasperng-nus) [inline src/test/java/seedu/address/storage/JsonInternBookStorageTest.java:67]: this as well, perhaps the name can be changed to JsonInternBookStorage

File: src/test/java/seedu/address/storage/StorageManagerTest.java [hunk 9]
```diff
@@ -54,7 +54,7 @@ public void addressBookReadSave() throws Exception {
          * {@link JsonAddressBookStorage} class.
          * More extensive testing of UserPref saving/reading is done in {@link JsonAddressBookStorageTest} class.
          */
-        InternBook original = getTypicalAddressBook();
+        InternBook original = getTypicalInternBook();
         storageManager.saveAddressBook(original);
```
Comment (jasperng-nus) [inline src/test/java/seedu/address/storage/StorageManagerTest.java:58]: in the storagemanger class perhaps you could change the method saveAddressBook to better fit our context

File: src/test/java/seedu/address/storage/StorageManagerTest.java [hunk 10]
```diff
@@ -54,7 +54,7 @@ public void addressBookReadSave() throws Exception {
...
         ReadOnlyInternBook retrieved = storageManager.readAddressBook().get();
```
Comment (jasperng-nus) [inline src/test/java/seedu/address/storage/StorageManagerTest.java:59]: sames go to the readAddressBook

**Review by jasperng-nus — COMMENTED** [review]
Overall LGTM on the whole, just a few instances of addressbook was missed out, we could probably split the work with everyone else too.

File: src/test/java/seedu/address/storage/JsonInternBookStorageTest.java [hunk 11]
```diff
@@ -64,7 +64,8 @@ public void readInternBook_invalidAndValidCompanyInternBook_throwDataLoadingExce
     public void readAndSaveInternBook_allInOrder_success() throws Exception {
         Path filePath = testFolder.resolve("TempAddressBook.json");
         InternBook original = getTypicalInternBook();
-        JsonAddressBookStorage jsonAddressBookStorage = new JsonAddressBookStorage(filePath);
+
+        JsonInternBookStorage jsonAddressBookStorage = new JsonInternBookStorage(filePath);
```
Comment (jasperng-nus) [inline src/test/java/seedu/address/storage/JsonInternBookStorageTest.java:68]: the name here i think can be change to jsonInternBookStorage

**Review by jasperng-nus — COMMENTED** [review]
LGTM! except for a small naming issue with regard to our context

**Pull request merged**